Cargo.toml

```diff
@@ -19,8 +20,7 @@ manage_clipboard = ["clipboard", "thread_local"]
 open_url = ["webbrowser"]
 
 [dependencies]
-bevy = { version = "0.4", default-features = false, features = ["render"] }
-bevy_winit = "0.4.0"
+bevy = { git = "https://github.com/bevyengine/bevy", branch = "main", default-features = false, features = ["render", "bevy_winit"] }
```

Comment:
Would we still want bevy master, or bevy 0.5 now that it's released?

Reply:
Yep, I've changed it now